apps/site/lib/site_web/controllers/schedule/line/helpers.ex

```diff
@@ -80,10 +80,36 @@ defmodule SiteWeb.ScheduleController.Line.Helpers do
   defp do_get_branch_route_stops(route, direction_id, route_pattern_id) do
     route.id
     |> get_route_patterns(direction_id, route_pattern_id)
-    |> Enum.filter(&by_typicality(&1, route_pattern_id))
+    |> filter_by_min_typicality()
```

Comment:
I'm wondering if it might make sense to account for the ferry here? Like what if we did
```
defp filter_by_min_typicality(route_patterns, route_pattern_id, route)
  if the route is a ferry do
     # invoke temporary_hh_ferry_only_func here
  else
    # proceed normally
     Enum.filter(route_patterns, &by_typicality(&1, route_pattern_id))
  end
end)
```

Reply:
Ohhhh, for other routes besides ferry, we should be filtering by typicality == 1?

Reply:
I'm mostly hesitant to change how route patterns are selected for _every_ single route, especially if we think the problem we are fixing is pretty isolated to this ferry.. 😬

Reply:
Ok!  I'll account for this `if-else` in the parent method then, so that "filter_by_min_typicality" only does one type of action